src/main/java/com/microsoftopentechnologies/windowsazurestorage/helper/CredentialMigration.java

```diff
@@ -68,7 +68,10 @@ protected static List<StorageAccountInfo> getOldStorageConfig(
                 String blobURL = elem.getElementsByTagName("blobEndPointURL").item(0)
                         .getChildNodes().item(0).getNodeValue();
 
-                storages.add(new StorageAccountInfo(accName, accKey, blobURL));
+                String cdnURL = elem.getElementsByTagName("cdnEndPointURL").item(0)
```

Comment:
no need to touch this class as it's for an old migration, theres no way this would be in the xml file

Reply:
Addressed in latest commits.